code/__HELPERS/icon_smoothing.dm

```diff
@@ -385,12 +377,10 @@
 
 //SSicon_smooth
 /proc/queue_smooth(atom/A)
-	if(!A.smooth || A.smooth & SMOOTH_QUEUED)
+	if(!A.smooth)
 		return
 
-	SSicon_smooth.smooth_queue += A
-	SSicon_smooth.can_fire = 1
-	A.smooth |= SMOOTH_QUEUED
+	SSicon_smooth.add_to_queue(A)
```

Comment:
I think this might be worth a define since these are called a lot at mapload.

Reply:
Turned it into a define.